config/integration/apps.yml

```diff
@@ -59,26 +59,9 @@ linear:
   id: linear
   logo: linear.png
   i18n_key: linear
-  action: /linear
+  action: https://linear.app/oauth/authorize
```

Comment:
Not sure if hardcoding this is a great idea. People on the `enterprise`  plan with linear might have a different domain. Anyway this is not something that might hit us now. So if it requires significant changes. We can ignore this feedback for now

CC: @sojan-official

Reply:
Makes sense. I'll add a comment there and revisit later.